NXP-29229 explorer templates for deployment (#5)
nuxeo/nuxeo-explorer: ReadMe.md

 - check that a given contribution is deployed
 - play with Nuxeo Runtime

## Configuration

The template `explorer-sitemode` enables the nuxeo.conf property `org.nuxeo.apidoc.site.mode` and
defines an anonymous user.
The property `org.nuxeo.apidoc.site.mode` comes with a more user friendly design and hides the current
"live" distribution from display and API.

The template `explorer-virtualadmin` disables the usual `Administrator` user creation at database

> initialization and add a virtual admin user with name `apidocAdmin`, which password can be changed using

Reviewer: ```suggestion
initialization and adds a virtual admin user with name `apidocAdmin`, whose password can be changed using
```
